core/chains/solana/soltxm/txm.go

```diff
@@ -15,7 +20,13 @@ import (
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
-const MaxQueueLen = 1000
+const (
+	MaxQueueLen       = 1000
+	MaxRetryTimeMs    = 500              // max tx retry time
+	MaxRetryTxTimeout = 5 * time.Second  // transaction timeout to stop retrying
+	MaxSigsToConfirm  = 256              // max number of signatures in GetSignatureStatus call
+	MaxConfirmTimeout = 15 * time.Second // confirmation timeout to stop checking signature
```

Comment:
better to make `MaxRetryTxTimeout` and `MaxConfirmTimeout` configurable? maybe as env vars as it would likely be a network level config?

Reply:
We should have a consistent way to configure everything, so any modifiable configuration should be part of the Chains configuration (if mutable, otherwise an in-line constant is fine).

That way a user has a consistent way (CLI) to surface, learn about, and set configs.

Reply:
adding the new configuration variables: https://github.com/smartcontractkit/chainlink-solana/pull/327